src/app/components/core/control-dispatcher/api-flow/add-edit-api-flow/add-edit-api-flow.component.html

```diff
@@ -253,12 +260,19 @@
                                                     <label class="label lbl-child has-text-left">Constants</label>
                                                 </div>
                                                 <div class="field-body">
-                                                    <button class="button px-0 pt-1 is-small is-link is-light has-background-white" (click)="addValueControl('constants')">
-                                                        <span class="icon is-small pt-1 mr-0">
-                                                            <i class="fa fa-xs fa-plus" aria-hidden="true"></i>
-                                                        </span>
-                                                        <span>Add constant</span>
-                                                    </button>
+                                                    <ng-container *ngIf="rolesService.hasControlAccess(); else NoConstant">
+                                                        <button class="button px-0 pt-1 is-small is-link is-light has-background-white" (click)="addValueControl('constants')">
+                                                            <span class="icon is-small pt-1 mr-0">
+                                                                <i class="fa fa-xs fa-plus" aria-hidden="true"></i>
+                                                            </span>
+                                                            <span>Add constant</span>
+                                                        </button>
+                                                    </ng-container>
+                                                    <ng-template #NoConstant>
```

Comment:
Why do we need this?

Reply:
This condition is added to show `None` to less privileged user if no constants are available.